code/game/objects/items/scanners.dm

```diff
@@ -162,7 +162,7 @@ REAGENT SCANNER
 		"oxy" = round(patient.getOxyLoss()),
 		"clone" = round(patient.getCloneLoss()),
 
-		"revivable" = patient.getBruteLoss() + patient.getFireLoss() + patient.getToxLoss() + patient.getOxyLoss() + patient.getCloneLoss() <= 200,
+		"revivable" = patient.getBruteLoss() + patient.getFireLoss() + patient.getToxLoss() + patient.getCloneLoss() <= 215, // around the revive threshold for people with corpsman level skills
```

Comment:
Stop hard coding this, set it to a formula that pulls from relevant data.
If medical skill ever changes, this is no longer valid
If defib healing changes, this is no longer valid.
If revivable threshold changes, this is no longer valid.

Reply:
yeah that's stinky. this is also assuming the maxhp never changes as well.